static/app/components/hotkeys.tsx

```diff
@@ -0,0 +1,121 @@
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+import {getKeyCode} from 'sentry/utils/getKeyCode';
+
+const macModifiers = {
+  18: '⌥',
+  17: '⌃',
+  91: '⌘',
+};
+
+const normalModifiers = {
+  18: 'ALT',
+  17: 'CTRL',
+  91: 'CMD',
+};
+
+const genericGlyphs = {
+  16: '⇧',
+  8: '⌫',
+  37: '←',
+  38: '↑',
+  39: '→',
+  40: '↓',
+  9999: '+',
+};
+
+const keyToDisplay = (
+  key: string,
+  platform?: 'macos' | string
```

Comment:
Why is this part configurable?

maybe the parameter should be called `forcePlatform` and its type is `'mac' | 'win'`?

Reply:
or maybe instead of `'win'` it's `'generic'`

Reply:
If you wanted to force render the fallback, was using it for testing purposes, renaming it makes sense.